docs/_static/.gitignore

```diff
@@ -0,0 +1 @@
+# Empty directory
```

Comment:
Why do we need to keep this directory. When building docs it should be automatically created.